crates/matrix-sdk/src/media.rs

```diff
@@ -437,17 +437,17 @@ impl Media {
     }
 
     /// Upload the file bytes in `data` and construct an attachment
-    /// message with `body`, `content_type`, `info` and `thumbnail`.
+    /// message with `body`, `content_type`, `info` and `thumbnail`,
+    /// optionally with `formatted` and `filename`.
```

Comment:
This comment is stale too, since we're passing an `AttachmentConfig` now :)

Reply:
These two comments refers to the return value, which will contain the new fields

Reply:
I think the previous version mentioned all the parameters names, including `body` which has been renamed to `filename` and should be updated here too, so I'm holding onto my previous comment. (That being said, I don't think each method should repeat all the names of all parameters, it might not be quite useful without adding some description, so feel free to remove everything after (including) `with body`.)

Reply:
Yeah it's a bit redundant.